docs/docs/API-Reference/api-monitor.md

```diff
@@ -670,8 +670,4 @@ curl -X GET \
 ```
 
   </TabItem>
-</Tabs>
-
-## See also
-
-- [Session ID](/session-id)
\ No newline at end of file
+</Tabs>
```

Comment:
What was the reasoning behind removing this link? Does the monitor endpoint use a different session_id?